<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxInfoWidget</class>
 <widget class="QWidget" name="BoxInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>100</y>
     <width>251</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>480</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开货柜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>288</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ce5c00;&quot;&gt;货柜信息平面展示图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>191</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ce5c00;&quot;&gt;所选货柜信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0c161c;&quot;&gt;车右&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0c161c;&quot;&gt;车前&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0c161c;&quot;&gt;车左&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0c161c;&quot;&gt;车后&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>671</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_17">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>96</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B2_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B2_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B2_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B2_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B2_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B2_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>97</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/C3_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/C3_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/C3_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/C3_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/C3_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/C3_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>C3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_16">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>97</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B7_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B7_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B7_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B7_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B7_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B7_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_15">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>97</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B5_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B5_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B5_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B5_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B5_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B5_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_14">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>97</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B3_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B3_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B3_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B3_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B3_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B3_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>97</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B1_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B1_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B1_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B1_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B1_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B1_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>97</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A7_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A7_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A7_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A7_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A7_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A7_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>96</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A8_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A8_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A8_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A8_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A8_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A8_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A8</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>96</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A6_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A6_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A6_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A6_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A6_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A6_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>96</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A4_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A4_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A4_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A4_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A4_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A4_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_20">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>96</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B8_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B8_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B8_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B8_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B8_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B8_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>96</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A2_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A2_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A2_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A2_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A2_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A2_6.png);
}
QRadioButton:pressed
{ padding-left:1px;
  padding-top:1px;}</string>
   </property>
   <property name="text">
    <string>A2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>97</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A3_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A3_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A3_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A3_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A3_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A3_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>97</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A5_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A5_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A5_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A5_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A5_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A5_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_19">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>96</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B6_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B6_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B6_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B6_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B6_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B6_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_18">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>96</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/B4_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/B4_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/B4_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/B4_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/B4_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/B4_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>B4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>97</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/C1_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/C1_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/C1_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/C1_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/C1_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/C1_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>96</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/C4_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/C4_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/C4_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/C4_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/C4_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/C4_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>C4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>96</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/C2_3.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/C2_2.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/C2_1.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/C2_1.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/C2_1.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/C2_3.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>97</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton：
QRadioButton::indicator 
{
    width: 13px;
    height: 13px;
}
QRadioButton::indicator::unchecked 
{  
    image: url(:/images/A1_6.png);
}
  
QRadioButton::indicator:unchecked:hover 
{   
    image: url(:/images/A1_5.png);
}
  
QRadioButton::indicator:unchecked:pressed 
{  
    image: url(:/images/A1_4.png);
}
  
QRadioButton::indicator::checked 
{   
    image: url(:/images/A1_4.png);
}
QRadioButton::indicator:checked:hover 
{  
    image: url(:/images/A1_4.png);
}
  
QRadioButton::indicator:checked:pressed 
{ 
    image: url(:/images/A1_6.png);
}
QRadioButton:pressed
{ padding-left:2px;
  padding-top:2px;}</string>
   </property>
   <property name="text">
    <string>A1</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>291</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <zorder>graphicsView_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>textBrowser</zorder>
  <zorder>pushButton_openBox</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>radioButton_17</zorder>
  <zorder>radioButton_10</zorder>
  <zorder>radioButton_16</zorder>
  <zorder>radioButton_15</zorder>
  <zorder>radioButton_14</zorder>
  <zorder>radioButton_13</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>radioButton_8</zorder>
  <zorder>radioButton_7</zorder>
  <zorder>radioButton_6</zorder>
  <zorder>radioButton_20</zorder>
  <zorder>radioButton_5</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_19</zorder>
  <zorder>radioButton_18</zorder>
  <zorder>radioButton_9</zorder>
  <zorder>radioButton_12</zorder>
  <zorder>radioButton_11</zorder>
  <zorder>radioButton_1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
